python/lsst/pipe/tasks/assembleCoadd.py

```diff
@@ -562,7 +573,7 @@ def backgroundMatching(self, inputData, refExpDataRef=None, refImageScaler=None)
                                imageScalerList=newScaleList, backgroundInfoList=newBackgroundStructList)
 
     def assemble(self, skyInfo, tempExpRefList, imageScalerList, weightList, bgInfoList=None,
-                 altMaskList=None, mask=None):
+                 altMaskList=None, mask=None, supplementaryData=None):
```

Comment:
Add `supplementaryData` to the documented `param[in]`'s

Reply:
Also, it's not actually used here. Is it only used if you retarget `assemble`?

Reply:
Yes thats right. If you're writing a new assembleCoadd subclass that needs extra input data that can be computed with the coadd dataRef, you'd override makeSupplementaryData().   In CompareWarpAssemble the template can eventually just be read off disk in  makeSupplementaryData when we have the capability to generate many different coadds, but for now we need to generate a PSF-Matched MEDIAN coadd on the fly. We can't reuse the results of prepareInputs from the main task (Like SafeClip) because those are for the direct warps (and don't apply to the psf-matched warps).  So makeSupplementaryData generates it by running the subtask.